sugaroid/brain/constants.py

```diff
@@ -163,7 +167,7 @@
 
 SUGAROID = ["🇸​🇺​🇬​🇦​🇷​🇴​🇮​🇩​", "sυgαяσι∂", "🅂🅄🄶🄰🅁🄾🄸🄳", "🆂🆄🅶🅰🆁🅾🅸🅳"]
 
-BYE = ["bye", "cya", "quit", "exit", "sleep", "sayonara", "annyeonghi gaseyo"]
+BYE = ["bye", "cya", "quit", "exit", "sleep", "sayonara", "annyeonghi gaseyo", "Au Revoir 🖐"]
```

Comment:
These are the responses which sugaroid accepts as bye responses from a user, not what sugaroid tells